setup.py

```diff
@@ -115,7 +115,7 @@
     "fugashi>=1.0",
     "GitPython<3.1.19",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.1.0,<1.0",
+    "huggingface-hub==0.8.0.rc2",
```

Comment:
This (and the change in the dependency version table) are isolated in a single commit that will need to be reverted before merging